### PR TITLE
Fix pytest benchmarks and shark_tank generation.

### PR DESCRIPTION
**Pull request by monorimet**
- fix setup_venv.sh for benchmarks/imports etc.
- unpin torch-mlir
- fix torch benchmarks in SharkBenchmarkRunner
- generate SD artifacts using `build_tools/stable_diffusion_testing.py` and `--import_mlir`
- decouple SD gen from `tank/generate_sharktank` for now

### CODE REVIEW

**Comment by monorimet** [comment]
https://github.com/openxla/iree/pull/14303 updates some experimental TF `mlir` API imports but with current pypi versions of tf-nightly the API does not match. Switching tf model generation off for now.

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
     install_requires=[
         "numpy",
         "PyYAML",
-        "torch-mlir",
+        "torch-mlir==20230802.121",
```
Comment (powderluv) [inline setup.py:42]: wait wait why

Comment (monorimet) [reply setup.py:42]: Just testing, it wouldn't pull the newest and was instead pulling from earlier today.

Comment (monorimet) [reply setup.py:42]: We don't need the pin when merging

File: .github/workflows/test-models.yml [hunk 2]
```diff
@@ -157,7 +157,7 @@ jobs:
     - name: Validate Stable Diffusion Models (Windows)
       if: matrix.suite == 'vulkan' && matrix.os == '7950x'
       run: |
-        ./setup_venv.ps1
+        ./setup_venv.ps1 --force
```
Comment (powderluv) [inline .github/workflows/test-models.yml:160]: shouldn't need this. This should be a new venv ?

File: setup.py (hunk 1 above)
Comment (monorimet) [reply setup.py]: Resolving as this pin was removed.

File: .github/workflows/test-models.yml (hunk 2 above)
Comment (monorimet) [reply .github/workflows/test-models.yml]: Makes sense -- I think I was trying to get the right torch-mlir version but it was a different issue.
Removed this change.

File: shark/shark_benchmark_runner.py [hunk 3]
```diff
@@ -124,42 +124,48 @@ def benchmark_frontend(self, modelname):
...
+            frontend_model.cpu()
+            input.cpu()
         # TODO: re-enable as soon as pytorch CUDA context issues are resolved
-        try:
-            frontend_model = torch.compile(
-                frontend_model, mode="max-autotune", backend="inductor"
-            )
-        except RuntimeError:
-            frontend_model = HFmodel.model
+        # try:
```
Comment (dan-garvey) [inline shark/shark_benchmark_runner.py:152]: dead code?

**Review by dan-garvey — DISMISSED** [review]

File: shark/shark_benchmark_runner.py (hunk 3 above)
Comment (monorimet) [reply shark/shark_benchmark_runner.py:152]: Yeah even if we enable this again it probably will need updating -- getting GPU baselines with inductor in these pytests is quite a mess.

**Review by powderluv — APPROVED** [review]

**Pull request merged**